src/App.css

```diff
@@ -1,39 +1,30 @@
-.App {
-  text-align: center;
+#calculator {
+  width: 40%;
+  margin: 10vh 0 0 28%;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.cal-val {
+  width: 98%;
+  height: 2.5rem;
+  background-color: rgb(187, 187, 187);
 }
```

Comment:
- [ ] You can have text-align property here to align the input text to the right like the original design. This is a business requirement and you need to stick to it. 
- [ ] You can also set the color property here to make the input text just like the original design.
- [ ] As you can compare your design with the original design, the division operator needs to be changed. 
![image](https://user-images.githubusercontent.com/64894927/129628722-9dc53f74-6859-4378-9f10-801971693e7c.png)
![image](https://user-images.githubusercontent.com/64894927/129628764-554e0b71-a2ee-49af-98e0-bebac7f3f0d5.png)